### PR TITLE
Add `carto.basemaps` constants for easier use of CARTO styles

### PR DESCRIPTION
**Pull request by VictorVelarde**
### Related to https://github.com/CartoDB/carto-vl/issues/905

New available constants (public API):
* `carto.basemaps.voyager`
* `carto.basemaps.darkmatter`
* `carto.basemaps.positron`

So now you can create a map like this:
```js
const map = new mapboxgl.Map({
      container: 'map',
      style: carto.basemaps.voyager,
      center: [0, 0],
      zoom: 1,
      scrollZoom: false,
      dragRotate: false,
      touchZoomRotate: false,
    });
````

I have updated the **examples**, so no urls are spread all over the project

### CODE REVIEW

**Comment by VictorVelarde** [comment]
@elena I have also fixed some relative paths on examples/advanced/landin-page/*.html files

**Comment by VictorVelarde** [comment]
@davidmanzanares There are 3 failing render tests. I tried to generate those in my Mac, but then those failed at CI.

File: examples/advanced/landing-page/rivers.html [hunk 1]
```diff
@@ -1,17 +1,19 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <title>US Population by County | CARTO</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta charset="UTF-8">
   <!-- Include CARTO VL JS -->
-  <script src="../../dist/carto-vl.js"></script>
+  <script src="../../../dist/carto-vl.js"></script>
```
Comment (davidmanzanares) [inline examples/advanced/landing-page/rivers.html]: was this broken?

Comment (VictorVelarde) [reply examples/advanced/landing-page/rivers.html]: @elenatorro is working in this new landing-page thing (a new folder has just been added)

**Comment by VictorVelarde** [comment]
Thx 👍

**Pull request merged**